.github/workflows/compilation-checks.yml

```diff
@@ -19,15 +19,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arch: [ARM, ARM_HYP, AARCH64, RISCV64, X64]
+        arch: [ARM, ARM_HYP, AARCH64, RISCV64, X64, Loongarch64]
         compiler: [gcc, llvm]
         exclude:
           # llvm RISCV64 compilation is not currently supported
           - arch: RISCV64
             compiler: llvm
+          # llvm Loongarch64 compilation is not currently supported
+          - arch: Loongarch64
+            compiler: llvm
     steps:
-    - uses: actions/checkout@v3
-    - uses: seL4/ci-actions/standalone-kernel@master
+    - uses: actions/checkout@v2
+    - uses: tyyteam/la-seL4-ci-actions/standalone-kernel@master
```

Comment:
We cannot approve workflow runs with actions from other repositories. Please revert those change before we look closer at this PR (also in cparser.yml and push.yml).